proto/eth/v1alpha1/slasher.proto

```diff
@@ -29,10 +29,21 @@ service Slasher {
     // Subscription to receive all slashable attester slashing events found by the watchtower.
     rpc SlashableAttestations(google.protobuf.Empty) returns (stream ethereum.eth.v1alpha1.AttesterSlashing);
 }
+
+message ValidatorIDToIdxAtt {
+    repeated uint64 indices = 1 ;
+    bytes data_root = 2;
+    // 96 bytes aggregate signature.
+    bytes signature = 3;
+}
+message ValidatorIDToIdxAttLst {
```

Comment:
Is this a typo? Do you mean to spell `List` not `Lst`?

Reply:
it was part of an effort to shorten the var name https://github.com/prysmaticlabs/prysm/pull/3322#discussion_r335874295

Reply:
chnaged